tokio-console/src/config.rs

```diff
@@ -48,6 +50,26 @@ pub struct Config {
     #[clap(long = "log", env = "RUST_LOG")]
     log_filter: Option<LogFilter>,
 
+    /// Enable lint warnings.
+    ///
+    /// This is a comma-separated list of warnings to enable.
+    ///
+    /// Each warning is specified by its name, which is one of:
+    ///
+    /// * `self-wakes` -- Warns when a task wakes itself more than a certain percentage of its total wakeups.
+    ///
+    /// * `lost-waker` -- Warns when a task is dropped without being woken.
+    ///
+    /// * `never-yielded` -- Warns when a task has never yielded.
+    ///
+    #[clap(long = "warn", short = 'W', value_delimiter = ',', num_args = 1..)]
+    #[clap(default_values_t = vec![
+        KnownWarnings::SelfWakes,
+        KnownWarnings::LostWaker,
+        KnownWarnings::NeverYielded
+    ])]
+    pub(crate) warns: Vec<KnownWarnings>,
```

Comment:
let's rename the field to 
```suggestion
    pub(crate) warnings: Vec<KnownWarnings>,
```

as that's the correct plural. also, in the help text, the argument should be displayed as `--warn <WARNINGS>` rather than `--warn <WARNS>`, which i think is a bit nicer.

Reply:
Updated.